source/crud/crud.rst

```diff
@@ -741,30 +742,37 @@ older sharded clusters.
 estimatedDocumentCount
 ~~~~~~~~~~~~~~~~~~~~~~
 
-On server versions greater than or equal to 4.9.0 (wire version 12 or higher),
-the estimatedDocumentCount function is implemented using the ``$collStats``
-aggregate pipeline stage with ``$group`` to gather results from multiple shards.
-As documented above, the only supported option is maxTimeMS::
-
-  pipeline = [
-    { '$collStats': { 'count': {} } },
-    { '$group': { '_id': 1, 'n': { '$sum': '$count' } } }
-  ]
-
-Similar to the count command, the estimated count of documents is returned
-in the ``n`` field. Implementations can assume that the document containing
-the single result of the aggregation pipeline is contained in the first batch of
-the server's reply to the aggregate command. It is not necessary to execute a getMore
-operation to ensure that the result is available.
-
-In the event this aggregation is run against a non-existent namespace, a NamespaceNotFound(26)
-error will be returned during execution. Drivers MUST interpret the server error code 26 as
-a ``0`` count.
-
-For server versions less than 4.9.0 (wire version 11 or under), the estimatedDocumentCount
-function is implemented using the ``count`` command with no query filter, skip,
-limit, or other options that would alter the results. Once again, the only supported
-option is maxTimeMS.
+The estimatedDocumentCount function is implemented using the ``count`` command
+with no query filter, skip, limit, or other options that would alter the
+results. Once again, the only supported option is maxTimeMS.
+
+Drivers MUST document that, due to a bug in versions 5.0.0-5.0.7 of MongoDB,
+estimatedDocumentCount was not included in v1 of the Stable API, so users of the
+Stable API are recommended to upgrade their server version to 5.0.8+ or set
+``apiStrict: false`` to avoid encountering errors when using
+estimatedDocumentCount.
```

Comment:
"Bug" is rather strong wording. `count` was intentionally omitted from the Stable API. It was an oversight that this would break estimatedDocumentCounts on views. I'm not sure how best to word this, but am open to suggestions.

Reply:
I took a second pass at this, let me know what you think. I also added a section to the Q&A about why it was decided that this change doesn't require a SemVer major version.